src/PatientBrowserTutorial/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
```

Comment:
I would like to get rid of this custom style sheet. This sample is not about style and you have added bootstrap, so do one or the other, but not both.